aptos-move/framework/aptos-framework/sources/primary_fungible_store.move

```diff
@@ -132,6 +147,31 @@ module aptos_framework::primary_fungible_store {
         }
     }
 
+    inline fun apt_store_address(account: address): address {
+        if (features::primary_apt_fungible_store_at_user_address_enabled()) {
+            account
+        } else {
+            object::create_user_derived_object_address(account, @aptos_fungible_asset)
+        }
+    }
+
+    public(friend) fun is_apt_balance_at_least(account: address, amount: u64): bool {
```

Comment:
Doesn't feel like any apt-specific logic should be in PFS? Can this be in aptos_account?

Reply:
we are specializing APT PFS to be more efficient, so these are all variants of the same functions defined in this file. (so is easier to look and see)

but I can create aptos_primary_fungible_store.move if you prefer to move them out

Reply:
moves to aptos_primary_fungible_store.move. 

let me know if apt_primary_fungible_store name (i.e. aptos -> apt) makes more sense

Reply:
Yeah apt makes more sense